projects/python-client/pyproject.toml

```diff
@@ -90,6 +90,7 @@ stringcase = "^1.2.0"
 tabulate = "^0.8.0"
 toolz = ">=0.11.0,<0.13.0"
 validators = ">=0.18.0,<0.21.0"
+ipynbname = "^2021.3.2"
```

Comment:
is `ipynbname` packaged on conda? (I'm assuming so!)

Reply:
Here it is https://anaconda.org/conda-forge/ipynbname